packages/toolpad-app/typings/@auth.d.ts

```diff
@@ -0,0 +1,5 @@
+export declare module '@auth/core/types' {
+  interface Profile {
+    verifiedEmails?: string[];
```

Comment:
If this isn't public API, should we really rely on it?

Reply:
With your suggestion `verifiedEmails` shouldn't be needed anymore so I'll remove it, it was only for verifying the restricted domains with the Github provider.